tests/integration/targets/docker_image/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+# Copyright 2022 Red Hat | Ansible
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# Used to create an alternate tag of an image and verify the archive will re-generate even with same image hash/id.
+docker_test_image_hello_world_archive_alt: docker_test_image_hello_world:latest
```

Comment:
Why use a fixed name and not generate one (like `iname` and `iname_1`)? That way the name should be unique per test, and potential leftovers from previous runs do not cause problems.

Reply:
The other examples I saw used fixed names, e.g. `tests/integration/targets/setup_docker_registry/vars/main.yml` (granted, those are referencing external images).

How do I auto-generate a name?

Reply:
Check out lines 8 and 9 in that file. If you want extra randomness add something like `(10000 | random)`.

Reply:
I moved it. I had to add an explict tag suffix because some tasks were interpreting it using `:latest` and others interpreted it using `:hello-world`